_rules/html-page-lang-b5c3f8.md

```diff
@@ -61,31 +61,31 @@ The `lang` attribute specified has a non-empty value.
 
 #### Failed Example 1
 
-There is no `lang` attribute specified.
+This `html` element does not have a `lang` attribute.
```

Comment:
Instead of saying what it doesn't have, should we say what it should have? For example:

```suggestion
This `html` element should have a `lang` attribute.
```

Reply:
That wouldn't match the "[meets / does not meets condition]" pattern that we suggest in #834, I think.